code/modules/mob/living/carbon/xenomorph/Castes/Defiler.dm

```diff
@@ -0,0 +1,198 @@
+	plasma_max = 550
+	plasma_gain = 19
+
+	// *** Health *** //
+	max_health = 290
+
+	// *** Evolution *** //
+	upgrade_threshold = 1600
+
+	// *** Defense *** //
+	armor_deflection = 47
+
+	// *** Defiler Abilities *** //
+	neuro_claws_dose = 6.7
+
+/datum/xeno_caste/defiler/ancient
+	upgrade_name = "Ancient"
+	caste_desc = "Being within mere eyeshot of this hulking, dripping monstrosity fills you with a deep, unshakeable sense of unease."
+	ancient_message = "You are the ultimate alien impregnator. You will infect the marines, see them burst open before you, and hear the gleeful screes of your larvae."
+	upgrade = 3
+
+	// *** Melee Attacks *** //
+	melee_damage_lower = 45
+	melee_damage_upper = 50
+
+	// *** Tackle *** //
+	tackle_damage = 55
+
+	// *** Speed *** //
+	speed = -1.2
+
+	// *** Plasma *** //
+	plasma_max = 575
+	plasma_gain = 20
+
+	// *** Health *** //
+	max_health = 300
+
+	// *** Evolution *** //
+	upgrade_threshold = 1600
+
+	// *** Defense *** //
+	armor_deflection = 50
+
+	// *** Defiler Abilities *** //
+	neuro_claws_dose = 7
+
+/mob/living/carbon/Xenomorph/Defiler
+	caste_base_type = /mob/living/carbon/Xenomorph/Defiler
+	name = "Defiler"
+	desc = "A large, powerfully muscled xeno replete with dripping spines and gas leaking dorsal vents."
+	icon = 'icons/Xeno/2x2_Xenos.dmi'
+	health = 225
+	maxHealth = 225
+	plasma_stored = 400
+	speed = -1
+	pixel_x = -16
+	old_x = -16
+	wound_type = "defiler" //used to match appropriate wound overlays
+	tier = 3
+	upgrade = 0
+	var/datum/effect_system/smoke_spread/smoke_system = null
+	var/last_defiler_sting = null
+	var/last_emit_neurogas = null
+	var/last_use_neuroclaws = null
+	var/neuro_claws = FALSE
+	var/emitting_gas = FALSE
+	var/neuro_claws_dose = DEFILER_CLAW_AMOUNT
+	actions = list(
+		/datum/action/xeno_action/xeno_resting,
+		/datum/action/xeno_action/regurgitate,
+		/datum/action/xeno_action/plant_weeds,
+		/datum/action/xeno_action/activable/emit_neurogas,
+		/datum/action/xeno_action/activable/defiler_sting,
+		/datum/action/xeno_action/neuroclaws,
+		)
+	inherent_verbs = list(
+		/mob/living/carbon/Xenomorph/proc/vent_crawl,
+		)
+
+/mob/living/carbon/Xenomorph/Defiler/update_icons()
+	if (stat == DEAD)
+		icon_state = "Defiler Dead"
+	else if (lying)
+		if ((resting || sleeping) && (!knocked_down && !knocked_out && health > 0))
+			icon_state = "Defiler Sleeping"
+		else
+			icon_state = "Defiler Knocked Down"
+	else
+		if (m_intent == MOVE_INTENT_RUN)
+			icon_state = "Defiler Running"
+		else
+			icon_state = "Defiler Walking"
+
+	update_fire() //the fire overlay depends on the xeno's stance, so we must update it.
+	update_wounds()
+
+/mob/living/carbon/Xenomorph/Defiler/throw_item(atom/target)
+	throw_mode_off()
+
+/mob/living/carbon/Xenomorph/Defiler/hitby(atom/movable/AM as mob|obj,var/speed = 5)
+	if(ishuman(AM))
+		return
+	..()
+
+/mob/living/carbon/Xenomorph/Defiler/Bumped(atom/movable/AM as mob|obj)
+	if(emitting_gas) //We don't get bumped around

```

Comment:
Even by tanks?

Reply:
Isn't there a tank_act or something

Reply:
Ah, the tank handles it through `/obj/vehicle/multitile/hitbox/cm_armored/Bump(var/atom/A)`, but it doesn't really call for `Bumped()` so this won't be an issue.